components/audioinput/client.go

```diff
@@ -45,7 +46,7 @@ func NewClientFromConn(ctx context.Context, conn rpc.ClientConn, name string, lo
 }
 
 func (c *client) Read(ctx context.Context) (wave.Audio, func(), error) {
-	stream, err := c.Stream(ctx)
+	stream, err := c.Stream(ctx, nil)
 	if err != nil {
 		return nil, nil, err
 	}
```

Comment:
Does `Read` also need the extra parameters fed to it?

Reply:
ya I think so